### ISSUE TITLE
[a11y] ♿ 'My Courses' UI should be more accessible

### ISSUE THREAD

**Issue by 9at8** [theRoughCode/WatsMyMajorBeta#21]
- Keyboard nav in dropdowns
- Enter to submit

stuff like that

![screenshot from 2018-07-25 15-51-26](https://user-images.githubusercontent.com/8235156/43224000-dd28a1c0-9022-11e8-9973-215aeec5001e.png)

**Comment by theRoughCode** [comment]
This should be fairly easy to implement with React DnD.
